mvhashmap_test.go

```diff
@@ -2,26 +2,174 @@ package block_stm
 
 import (
 	"fmt"
-	"github.com/stretchr/testify/require"
+	"math/rand"
 	"testing"
+
+	"github.com/stretchr/testify/require"
 )
 
+// create data for a given txIdx and incarnation
 func valueFor(txIdx, inc int) []byte {
 	return []byte(fmt.Sprintf("%ver:%ver:%ver", txIdx*5, txIdx+inc, inc*5))
 }
 
+// Result of the following benchmark test:
+// goos: darwin
+// goarch: arm64
+// pkg: github.com/paulgoleary/go-block-stm
+// BenchmarkWriteTimeSameLocationDifferentTxIdx-8   	       1	1383456542 ns/op	256013944 B/op	10000068 allocs/op
+// PASS
+// ok  	github.com/paulgoleary/go-block-stm	1.793s
+func BenchmarkWriteTimeSameLocationDifferentTxIdx(b *testing.B) {
+	mvh2 := MakeMVHashMap()
+	ap2 := []byte("/foo/b")
+	b.ResetTimer()
+	for i := 0; i < 1000000; i++ {
```

Comment:
It's a bit better to use b.N as the limit

Reply:
Sure, will do that. Thanks.